test/tests_for_sdkmr/EE2Runjob_test.py

```diff
@@ -730,11 +743,13 @@ def _set_up_common_return_values_batch(mocks):
     returned_parent_job = Job()
     returned_parent_job.id = ObjectId(_JOB_ID)
     returned_parent_job.user = _USER
-    mocks[SDKMethodRunner].save_and_return_job.return_value = returned_parent_job
-    mocks[Catalog].get_module_version.side_effect = [
-        {"git_commit_hash": _GIT_COMMIT_1},
-        {"git_commit_hash": _GIT_COMMIT_2},
+    mocks[CatalogCache].lookup_git_commit_version.side_effect = [
+        _GIT_COMMIT_1,
+        _GIT_COMMIT_2,
     ]
+    mocks[SDKMethodRunner].get_catalog_cache.return_value = mocks[CatalogCache]
```

Comment:
Again this line seems to be redundant to line 103

Reply:
done

Reply:
:+1: